Update google-ads-event-streaming.md (#72)
amplitude/amplitude-dev-center: docs/data/destinations/google-ads-event-streaming.md

1. Navigate back to the **Settings** tab and click **Edit**.
2. Toggle **Status** from **Disabled** to **Enabled**.
3. Save when finished.

## Common issues

### Stuck at the authentication step

> If you see `ERR_BLOCKED_BY_CLIENT` in your browser's console, ensure that your ad blocker is disabled, clear your browser's cache and cookies, and try to add the connection again.

Reviewer: ```suggestion
If you see `ERR_BLOCKED_BY_CLIENT` in your browser's console, disable your ad blocker, clear your browser's cache and cookies, and try to add the connection again.
```

### "Error: Invalid Customer ID"

Google Ads requires a **Customer ID** to configure as an integration with Amplitude. For more information, see [Find your Google Ads customer ID](https://support.google.com/google-ads/answer/1704344?hl=en) in Google's documentation.
Keep in mind, **Manager Account Customer ID** is different from the **Customer ID**.

### Insufficient permissions

Amplitude's Google Ads Event Streaming integration requires that your Google Ads Manager account has administrator privileges. This level of permission allows Amplitude to add and remove users from specific user lists in Google Ads.

> In the case of Google Ads, we add and remove users from a specific user list. As such, we need appropriate permissions to create and delete Google Ads account data.

Reviewer: ```suggestion
For Google Ads, Amplitude adds and removes users from a specific user list. As a result, Amplitude needs appropriate permissions to create and delete Google Ads account data.
```

1. Navigate back to the **Settings** tab and click **Edit**.
2. Toggle **Status** from **Disabled** to **Enabled**.
3. Save when finished.

> ## Common Issues

Reviewer: ```suggestion
## Common issues
```

For headings, we use sentence style capitalization.

### Stuck at the authentication step

> You might see the `ERR_BLOCKED_BY_CLIENT` error in your console, which is a common case when AdBlock is installed.

Reviewer: ```suggestion
If you see `ERR_BLOCKED_BY_CLIENT` in your browser's console, ensure that your ad blocker is disabled, clear your browser's cache and cookies, and try to add the connection again.
```
Reworded this a bit to remove the passive voice "is installed" and remove the word "please". In general, we want to write from the point of view that we tell users what to do, not ask them to do something.

> Please remove any browser extensions, clear your cache and cookies, and try to re-add the connection.

Reviewer: ```suggestion
```
See above

### "Error: Invalid Customer ID"

> A **Customer ID** is required to set up Google Ads. 

Reviewer: ```suggestion
```
Combined with next paragraph.

> You can learn more about your **Google Ads Customer ID** in [this article](https://support.google.com/google-ads/answer/1704344?hl=en)

Reviewer: ```suggestion
Google Ads requires a **Customer ID** to configure as an integration with Amplitude. For more information, see [Find your Google Ads customer ID](https://support.google.com/google-ads/answer/1704344?hl=en) in Google's documentation.
```

Change passive voice (is required) to active voice (requires). 

As a best practice when you link to other documentation, include the article name in the link text, and give the user an indication of where they are going, in this case "Google's documentation."

> - **Manager Account Customer ID** is different from the **Customer ID**.

Reviewer: ```suggestion
Keep in mind, **Manager Account Customer ID** is different from the **Customer ID**.
```
Lists with a single bullet don't need to be a list.

### Insufficient permissions

> Check if you have a Google Ads Manager account and you are an Admin.

Reviewer: ```suggestion
Amplitude's Google Ads Event Streaming integration requires that your Google Ads Manager account has administrator privileges. This level of permission allows Amplitude to add and remove users from specific user lists in Google Ads.
```
Avoid referring to Amplitude as "we." This can be especially confusing in documentation that covers a third-party integration like this, as the user may not know who "we" is.

In the case of Google Ads, we add and remove users from a specific user list. As such, we need appropriate permissions to create and delete Google Ads account data.

> You can learn more about it [here](https://support.google.com/google-ads/answer/9978556)  

Reviewer: ```suggestion
For more information, see [About access levels in your Google Ads Account
](https://support.google.com/google-ads/answer/9978556) in Google's documentation. 
```
